### PR TITLE
Lab3

### PR DESCRIPTION
**Pull request by artyom-mankevich**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            <header class="note-header">
+                <img class="note-header__icon"
+                     src="static/img/to-do-list.png" alt="List icon">
+                <h2 class="note-header__title">Plan activities</h2>
+            </header>
+            <p class="note__text">Create lists to break up your tasks and mark
+                accomplished goals</p>
+        </article>
+        <article class="note">
+            <header class="note-header">
```
Comment (katetushkan) [inline index.html]: there should be  only one header at the page.

File: templates/main.html [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+        <a class="header-auth__item" href="login.html">
+            <img class="header-logo__icon" src="../static/img/logout.svg"
+                 alt="Logout icon">
+            Log out
+        </a>
+    </section>
+</header>
+<section class="menu menu_hidden">
+    <section class="categories-menu categories-menu_hidden">
+        <h4 class="menu-title">Categories</h4>
```
Comment (katetushkan) [inline templates/main.html]: U can't use h4, if u have no h3, h2, h1 on ur page.

File: templates/main.html [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+                </header>
+                <section class="note-list">
+                    <div class="note-list__item">
+                        <img class="note-list__icon"
+                             src="../static/img/square.png"
+                             alt="Unchecked item">
+                        <p class="note-list__text">eggs</p>
+                    </div>
+                    <div class="note-list__item">
+                        <img class="note-list__icon"
```
Comment (katetushkan) [inline templates/main.html]: It should be checkbox

File: templates/signup.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+            <a class="header-logo__title" href="../index.html">To Do</a>
+        </header>
+        <section class="card-form">
+            <div class="card-form__item">
+                <label class="card__label" for="usernameInput">Username:</label>
+                <input id="usernameInput" class="card__input" type="text">
+            </div>
+            <div class="card-form__item">
+                <label class="card__label" for="emailInput">Email:</label>
+                <input id="emailInput" class="card__input" type="text">
```
Comment (katetushkan) [inline templates/signup.html]: make it type="email" browser will provide u additional validation

File: templates/main.html [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+                Notes
+            </button>
+            <button class="side-menu-categories__item">
+                <img class="side-menu-categories__icon"
+                     src="../static/img/bell.svg" alt="Reminders icon">
+                Reminders
+            </button>
+        </section>
+        <div class="side-menu-tags">
+            <h3 class="side-menu-tags__title">tags:</h3>
```
Comment (katetushkan) [inline templates/main.html]: U can't use h3 after h4

File: templates/main.html [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+    </button>
+    <section class="header-auth">
+        <a class="header-auth__item" href="login.html">
+            <img class="header-logo__icon" src="../static/img/logout.svg"
+                 alt="Logout icon">
+            Log out
+        </a>
+    </section>
+</header>
+<section class="menu menu_hidden">
```
Comment (katetushkan) [inline templates/main.html:35]: should be <nav>

File: templates/main.html [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+        <img class="menu-button__icon"
+             src="../static/img/menu.png" alt="Menu button">
+    </button>
+    <div class="header-logo">
+        <img class="header-logo__icon header-logo__icon_blue"
+             src="../static/img/check.svg"
+             alt="Logo icon">
+        <a class="header-logo__title" href="../index.html">To Do</a>
+    </div>
+    <button class="header__notification-button">
```
Comment (katetushkan) [inline templates/main.html:23]: If the main purpose of this is redirecting u to another page with notifications, than it's better to be a link^ not a button.

File: templates/signup.html [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+            <a class="card__submit-link" href="main.html">Sign up</a>
+        </section>
+        <div class="card-auth">
+            <p class="card-auth__text">Already have an account?</p>
+            <a class="card-auth__link" href="../templates/login.html">Log in</a>
+        </div>
+    </section>
+</main>
+</body>
+</html>
```
Comment (katetushkan) [inline templates/signup.html:63]: Screenshot 2022-03-28 at 12 27 17
small size does not look good(

File: index.html [hunk 9]
```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
```
Comment (katetushkan) [inline index.html:1]: I do mostly like how it looks, so I can give u 9 and recommend u to take a look at the comments I left and take care of them in ur future work if u are OK.

**Review by katetushkan — APPROVED** [review]

File: index.html (hunk 9 above)
Comment (katetushkan) [reply index.html:1]: U also have a possibility to have a 10 if u'll fix side menu, let me know ur decision.

Comment (artyom-mankevich) [reply index.html:1]: @katetushkan Yeah, I'll fix the side menu and other issues a bit later and notify you in this PR once it is done

File: templates/main.html (hunk 7 above)
Comment (artyom-mankevich) [reply templates/main.html:23]: There will be a pop-up window with notifications on the button press, so I think button tag fits

Comment (katetushkan) [reply templates/main.html:23]: okay then

File: index.html (hunk 9 above)
Comment (artyom-mankevich) [reply index.html:1]: @katetushkan fixed the issues you mentioned

**Review by katetushkan — CHANGES_REQUESTED** [review]
Upgraded

**Review by katetushkan — APPROVED** [review]

**Pull request merged**